### ISSUE TITLE
About Eq. (1.26) in section "Implicit methods"

### ISSUE THREAD

**Issue by astroboylrx** [Open-Astrophysics-Bookshelf/numerical_exercises#2]
I believe that Eq. (1.26) at [here](https://github.com/Open-Astrophysics-Bookshelf/numerical_exercises/blob/master/intro/intro.tex#L481) does not want expansions on time. I think the superscript of "t" on the right hand side should be changed from "n" to "n+1".
PS: I think the subscript of the Jacobin matrix, "0", may be a little confusing since the matrix uses digits (1-n) as the subscriptions of its elements. Maybe a few more explanations will make it clearer. Thanks.

**Comment by zingale** [comment]
you are right -- that's a typo (the n instead of n+1).  The proper superscript appears in 1.28.

I'll think about the notation issue as well.

**Comment by astroboylrx** [comment]
I found another two typos:
- [Eq. (3.11)](https://github.com/Open-Astrophysics-Bookshelf/numerical_exercises/blob/master/advection/advection.tex#L209) : I think the right-hand-side should be `1+C^2 \sin^2\theta`  
- [Exercise 4.1](https://github.com/Open-Astrophysics-Bookshelf/numerical_exercises/blob/master/burgers/burgers.tex#L205) : **to solver** => **to solve** 

> Extend your 1-d finite-volume solver for advection to solver Burgers' equation.

Sorry for my trivial comments on small things. :-)

**Comment by astroboylrx** [comment]
Maybe another typo in Exercise 3.10 at [here](https://github.com/Open-Astrophysics-Bookshelf/numerical_exercises/blob/master/advection/advection.tex#L822):

> However: you may find less than second-order _**is**_ your initial conditions have discontinuities and you are limiting.  

Maybe the original meaning is "you may find less second-order _**since**_ your initial conditions have discontinuities and you are limiting"?  
Thanks.

**Comment by astroboylrx** [comment]
About Figure 3.13 (tex code [here](https://github.com/Open-Astrophysics-Bookshelf/numerical_exercises/blob/master/advection/advection.tex#L829)): 
I couldn't reproduce it with minmod limiting method (the results are less than second order as described in [Exercise 3.10](https://github.com/Open-Astrophysics-Bookshelf/numerical_exercises/blob/master/advection/advection.tex#L815)). Later on, I found it was done by "centered" slope_type in your code, which means unlimited. I think the caption of Figure 3.13 is kind of misleading.

Thanks!

**Comment by zingale** [comment]
yes, you are right about FIgure 3.13.  I've explicitly added several different limiters to that plot now.

**Comment by astroboylrx** [comment]
Seems another typo in [Eq. (3.58)](https://github.com/Open-Astrophysics-Bookshelf/numerical_exercises/blob/master/advection/advection.tex#L944): 
I think the sign between the two fractions inside the square brackets should be plus:

```
a_{i,j}^{n+1} = a_{i,j}^n - \Delta t \left [
   \frac{(ua)_{i+\myhalf,j}^{n+\myhalf} - (ua)_{i-\myhalf,j}^{n+\myhalf}}{\Delta x} +
   \frac{(va)_{i,j+\myhalf}^{n+\myhalf} - (va)_{i,j-\myhalf}^{n+\myhalf}}{\Delta y} \right ]
```

Thank you! :-)

**Comment by zingale** [comment]
yes, you are correct
